bisqapps/shared/domain/src/commonMain/kotlin/network/bisq/mobile/domain/data/model/BtcFiatPrice.kt

```diff
@@ -0,0 +1,23 @@
+package network.bisq.mobile.domain.data.model
+
+open class BtcPrice: BaseModel() {
+
+    protected open val prices: Map<String, Double> = mapOf(
+        "USD" to 64000.50,
+        "EUR" to 58000.75,
+        "GBP" to 52000.30,
+    )
+
+    fun getBtcPrice(): Map<String, Double> {
```

Comment:
@nostrbuddha getters in kotlin code are discouraged, the compiler generates them automatically for each class with public fields

Reply:
Right. You mentioned it somewhere else already. Missed updating that here!
Will do it.